spec/api-process-spec.js

```diff
@@ -1,6 +1,13 @@
 const assert = require('assert')
 
 describe('process module', () => {
+  describe('process.getCreationTime()', () => {
+    it('returns a creation time', () => {
+      const creationTime = process.getCreationTime()
+      assert.equal(typeof creationTime, 'number')
```

Comment:
Please use `expect()` assertions from the [Chai library](http://www.chaijs.com/api/bdd/), which produce much better failure message than `assert`.

```js
const {expect} = require('chai')  // On the top of the file.
...
expect(creationTime).to.be.a('number').and.be.at.least(0)
```